### PR TITLE
make "Must Grade" and "Graded By" labels match actual backend validation

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** make "Must Grade" and "Graded By" labels match actual backend validation

JIRA: [EDUCATOR-5436](https://openedx.atlassian.net/browse/EDUCATOR-5436)

** What Changed ? **

- In the Edit ORA view in Studio, the tooltips for "Must Grade" and "Graded By" should now say that the value is 1-99 rather than 0-99

**Developer Checklist**

- [x] Reviewed the [release process](./release_process.md)
- [x] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
Again, not really worth the version bump, I'll just let it get picked up

**Review by nsprenkle — APPROVED** [review]
Visual review looks good 👍
